src/kaleidoscope/device/dygma/Raise.cpp

```diff
@@ -0,0 +1,599 @@
+/* -*- mode: c++ -*-
+ * kaleidoscope::device::dygma::Raise -- Kaleidoscope device plugin for Dygma Raise
+ * Copyright (C) 2017-2019  Keyboard.io, Inc
+ * Copyright (C) 2017-2019  Dygma Lab S.L.
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation, version 3.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifdef ARDUINO_SAMD_RAISE
+
+#include <Kaleidoscope.h>
+#include <Kaleidoscope-EEPROM-Settings.h>
+#include <Kaleidoscope-LEDControl.h>
+#include <KeyboardioHID.h>
+#include <Wire.h>
+
+#include "kaleidoscope/util/crc16.h"
+
+#define I2C_CLOCK_KHZ 200
+#define I2C_FLASH_CLOCK_KHZ 100 // flashing doesn't work reliably at higher clock speeds
+
+#define SIDE_POWER 1 // side power switch pa10
+
+#define LKEY 33 // raise::left_keys
+#define LPH LEDS_PER_HAND
+#define XX 0xFF // off
+
+const uint8_t PROGMEM gamma8[] = {
```

Comment:
I thought the gamma table was getting lifted out?

Reply:
Ow, it is! We just had two copies and I missed one.